.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: node_js
 node_js:
-  - 8.11
-before_install:
-  - npm i -g npm@6.2.0
+  - '10'
```

Comment:
@hacdias we don't support Node older than 10, probably thats why CI failed. After updating to 10, it is green now :+1:

Reply:
Oh, thanks for the info!